packages/neuron-wallet/src/controllers/wallets.ts

```diff
@@ -156,6 +161,32 @@ class WalletsController {
     }
   }
 
+  public static verifyPasswordComplexity = (password: string) => {
+    if (password.length < MIN_PASSWORD_LENGTH) {
+      throw Error(i18n.t('messages.wallet-password-less-than-min-length'))
```

Comment:
use the constant in the message means use the i18n interpolation, there's the [guide](https://www.i18next.com/translation-function/interpolation)

Reply:
Indeed.